packages/jira-adapter/src/filters/assets/assets_instances_deployment.ts

```diff
@@ -44,6 +44,15 @@ const filter: FilterCreator = ({ config }) => ({
       .forEach(instance => {
         instance.value.objectSchemaId = getParent(instance).value.id
       })
+
+    await awu(changes)
+      .filter(isInstanceChange)
+      .map(getChangeData)
```

Comment:
I think you should add it here as well
```suggestion
      .filter(isAdditionOrModificationChange)
      .map(getChangeData)
```